tools/integrations/manage.sh

```diff
@@ -198,7 +200,8 @@ cmd_publish() {
   local image_name; image_name=$(_get_docker_image_name "$path"/Dockerfile)
   local image_version; image_version=$(_get_docker_image_version "$path"/Dockerfile)
   local versioned_image=$image_name:$image_version
-  local latest_image=$image_name:latest
+  local latest_image="$image_name" # don't include ":latest", that's assumed here
+  local build_arch="linux/amd64,linux/arm64"
```

Comment:
Being in a comma sepped string feels a little odd to me, I wonder if we can change the code below this for the docker-compose build parts to take an array and produce the comma separated string rather than going the other way around.

Not really a big deal either way though

Reply:
I'm very 🤷 on this as well... I figured I would leave it as-is